config/prow/plugins.yaml

```diff
@@ -1313,6 +1319,11 @@ plugins:
     plugins:
     - override
 
+  kubernetes-sigs/kueue:
+    plugins:
+    - milestone
+    - milestonestatus
```

Comment:
did you remove the cherrypicker?

what's the milestonestatus for?

Reply:
cherrypicker is there, just enabled separately, see below